src/go/rdctl/pkg/snapshot/manager_test.go

```diff
@@ -167,6 +167,14 @@ func TestManager(t *testing.T) {
 		if len(snapshots) != 0 {
 			t.Fatalf("unexpected length of snapshots slice after delete %d", len(snapshots))
 		}
+		err = manager.Delete("no-such-snapshot-id")
+		if err == nil {
+			t.Errorf("Failed to complain when asked to delete a nonexistent snapshot")
+		}
+		err = manager.Restore("no-such-snapshot-id")
```

Comment:
This should definitely be a separate test case, since it has to do with `Restore` and not `Delete`.

Reply:
There's now a test-case on `Operations on nonexistent snapshots flag errors` . I did the above because it looked kind of costly calling `populateFiles()` when we don't need to actually use any of those files.